node/service/src/chain_spec.rs

```diff
@@ -833,6 +838,10 @@ fn rococo_staging_testnet_config_genesis(wasm_binary: &[u8]) -> rococo_runtime::
 				.chain(initial_authorities.iter().map(|x| (x.0.clone(), STASH)))
 				.collect(),
 		}),
+		pallet_beefy: Some(rococo_runtime::BeefyConfig {
+			// TODO [ToDr] Beefy authorities
```

Comment:
This should be initialized through the session interface with `on_genesis_session`.